x-pack/plugins/stack_connectors/public/connector_types/jira/jira_params.tsx

```diff
@@ -381,6 +383,51 @@ const JiraParamsFields: React.FunctionComponent<ActionParamsProps<JiraActionPara
             }
           )}
         />
+        <EuiFormRow
+          fullWidth
+          error={errors['subActionParams.incident.otherFields']}
+          isInvalid={
+            errors['subActionParams.incident.otherFields'] !== undefined &&
+            errors['subActionParams.incident.otherFields'].length > 0
+          }
+        >
+          <JsonEditorWithMessageVariables
+            messageVariables={messageVariables}
+            paramsProperty={'otherFields'}
+            inputTargetValue={actionParams.subActionParams?.incident.otherFields || undefined}
+            errors={errors.otherFields as string[]}
+            label={
+              <>
+                {i18n.translate('xpack.stackConnectors.components.jira.otherFieldsFieldLabel', {
+                  defaultMessage: 'Additional fields',
+                })}
+                <EuiIconTip
+                  size="s"
+                  color="subdued"
+                  type="questionInCircle"
+                  className="eui-alignTop"
+                  data-test-subj="otherFieldsHelpTooltip"
+                  aria-label={i18n.translate(
+                    'xpack.stackConnectors.components.jira.otherFieldsHelpTooltip',
+                    {
+                      defaultMessage: 'Additional fields help',
```

Comment:
Not sure but should this be the same as the message below?

Reply:
I would say no. AFAIK This is an alternative to the `?` icon, the screen reader will tell the user that there is this icon called "Additional fields help" and if they decide to click on it, the reader will read the tooltip message

Reply:
oh I see, thanks!